apps/faucet/src/App/App.tsx

```diff
@@ -133,14 +138,37 @@ export const App: React.FC = () => {
       now.getUTCHours(),
       now.getUTCMinutes()
     );
-    const startsAtToMilliseconds = startsAt * 1000;
-    if (nowUTC < startsAtToMilliseconds) {
-      setIsTestnetLive(false);
-    }
 
     // Fetch settings from faucet API
     (async () => {
       try {
+        let startsAtToMilliseconds = startsAt * 1000;
+        let startsAtText = defaults.startsAtText
+        try{
```

Comment:
It would be nice to move this `try...catch` block to the separete function. I think that would make code more readable. Also it should help with getting rid off mutations in multiple places. At the moment it's a bit difficult to follow what is going on.